### PR TITLE
Qp scaling tables

### PR DESCRIPTION
**Pull request by andrey-norkin**
Enabling lower bitrates to be achieved by SVT-AV1 when higher QP values are chosen. Some improvement can be seen on 8 and 10 bit content (0.3% - 0.7% BD-rate). 

Adjustment of QP values may need to be done since changes in QP values in the higher QP range would results in a more significant changes in BD-rate than what is currently observed.

The tables can be further optimized for performance in lower QP ranges.

### CODE REVIEW

File: Source/Lib/Common/Codec/EbRateControlProcess.c [hunk 1]
```diff
@@ -3244,76 +3245,375 @@ enum {
...
-        arfgf_low_motion_minq_10, arfgf_high_motion_minq_10,
-        inter_minq_10, rtc_minq_10, AOM_BITS_10);
-    init_minq_luts(kf_low_motion_minq_12, kf_high_motion_minq_12,
-        arfgf_low_motion_minq_12, arfgf_high_motion_minq_12,
-        inter_minq_12, rtc_minq_12, AOM_BITS_12);
-}
+static int gf_high = 2000;
+static int gf_low = 400;
+static int kf_high = 5000;
+static int kf_low = 400;
```
[the remark this answers is not in the document]
Comment (andrey-norkin) [reply Source/Lib/Common/Codec/EbRateControlProcess.c]: These have not changed, they existed in the old code. For some reason, Github marked them as changes.

**Comment by anaghdin** [comment]
looks good to me. Do you have the results over 30 short clips?

**Comment by andrey-norkin** [comment]
> looks good to me. Do you have the results over 30 short clips?

Yes, please see below, on 8 bits (-0.29% BD-rate on PSNR and -0.93 on VMAF), if evaluating with QP -q 20,32,43,55,59,63 (one extra QP is adding in the higher range to account for large quality steps in the higher QP range). 

Gains are somewhat higher on 10 bits, but the sequence set is different too.

Sequence | BDClassicPSNRY | BDVMAF
-- | -- | --
DOTA2_60f_420 | 0.377 | 0.313
KristenAndSara_1280x720_60f | -0.699 | -0.956
MINECRAFT_60f_420 | -1.758 | -5.066
Netflix_Aerial_1920x1080_60fps_8 | -0.772 | -3.268
Netflix_Boat_1920x1080_60fps_8bi | -0.956 | -2.553
Netflix_Crosswalk_1920x1080_60fp | -1.513 | -1.083
Netflix_DrivingPOV_1280x720_60fp | -0.374 | -1.949
Netflix_FoodMarket_1920x1080_60f | -0.977 | -1.711
Netflix_PierSeaside_1920x1080_60 | -0.474 | -0.145
Netflix_RollerCoaster_1280x720_6 | -0.432 | -1.567
Netflix_SquareAndTimelapse_1920x | -0.628 | -2.828
Netflix_TunnelFlag_1920x1080_60f | -1.401 | -5.911
STARCRAFT_60f_420 | 0.472 | 1.335
aspen_1080p_60f | -0.021 | 0.964
blue_sky_360p_60f | 0.221 | -0.644
dark720p_60f | 0.653 | 1.988
ducks_take_off_1080p50_60f | -0.498 | -1.925
gipsrestat720p_60f | 0.453 | 0.676
kirland360p_60f | 1.017 | 0.355
life_1080p30_60f | -0.216 | -1.51
niklas360p_60f | 0.072 | 0.626
red_kayak_360p_60f | -0.989 | -4.976
rush_hour_1080p25_60f | -2.216 | -1.767
shields_640x360_60f | 0.819 | 0.079
speed_bag_640x360_60f | 1.414 | 2.06
thaloundeskmtg360p_60f | 0.577 | 1.26
touchdown_pass_1080p_60f | -0.14 | -2.323
vidyo1_720p_60fps_60f | -0.44 | 1.375
vidyo4_720p_60fps_60f | -0.279 | 1.073
wikipedia_420 | -0.041 | 0.14
Average | -0.292 | -0.931

**Comment by kylophone** [comment]
Measured with AWCY: https://beta.arewecompressedyet.com/?job=%402019-10-15T20%3A51%3A27.590Z_qp_scaling&job=SVT-AV1-enc-mode-3%402019-10-06

**Comment by andrey-norkin** [comment]
> Measured with AWCY: https://beta.arewecompressedyet.com/?job=%402019-10-15T20%3A51%3A27.590Z_qp_scaling&job=SVT-AV1-enc-mode-3%402019-10-06

Should you apply the "reverse" to set the current SVT-AV1 as the "anchor"?

Best regards,
Andrey

**Comment by kylophone** [comment]
> > Measured with AWCY: https://beta.arewecompressedyet.com/?job=%402019-10-15T20%3A51%3A27.590Z_qp_scaling&job=SVT-AV1-enc-mode-3%402019-10-06
> 
> Should you apply the "reverse" to set the current SVT-AV1 as the "anchor"?
> 
> Best regards,
> Andrey

Yes.

**Comment by ultrawide** [comment]
Nice! I ran a test across 30 clips in context of M2 with QP values 20, 32, 43, 55 and also found an average y-psnr improvement of 0.29%.

**Comment by EwoutH** [comment]
@andrey-norkin Could you rebase your branch?

**Pull request merged**